packages/expo-audio/src/Audio.types.ts

```diff
@@ -21,54 +21,118 @@ export type AudioSource =
       headers?: Record<string, string>;
     };
 
+/**
+ * Represents an available audio input device for recording.
+ *
+ * This type describes audio input sources like built-in microphones, external microphones,
+ * or other audio input devices that can be used for recording. Each input has identifying
+ * information that can be used to select the preferred recording source.
+ */
 export type RecordingInput = {
+  /** Human-readable name of the audio input device. */
```

Comment:
```suggestion
  /** Human-readable name of the audio input device. */
```

Please add an example here (based on recent feedback on how strings often lack examples).

Reply:
I think that was strings we expect users to provide wasn't it? Like `urls` etc.? This is returned fromd the native side

Reply:
Makes sense. didn't know that this value was returned from native side.